### PR TITLE
Update to libvirt 5.6.0

### PR DESCRIPTION
**Pull request by rmohr**
**What this PR does / why we need it**:

Update kubevirt to libvirt 5.6.0.

* Libvirt 5.6.0 wants to set XATTRS when `remember_owner` is set to 1. Since we don't want to grant SYS_CAP_ADMIN privileges, disable `remember_owner`.
 * There is an issue with `dynamic_ownership`. It stopped working for us. Taking this as an opportuinity to clean up the code and set the ownership in a consistent fashion for all disks.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

`dynamic_ownership` should work in dependent of XATTRS. We will investigate this further, but for now just set the ownership of disks ourselves in a consistent way.

 https://github.com/kubevirt/libvirt/pull/39 prepared kubevirt/libvirt.

**Release note**:
<!--  Write your release note:

```release-note
Update to libvirt-5.6.0
```

### CODE REVIEW

**Comment by rmohr** [comment]
/test 

pull-kubevirt-e2e-okd-4.1
--

**Comment by rmohr** [comment]
@stu-gott @mhenriks should be ready for review now.

File: pkg/ephemeral-disk-utils/utils.go [hunk 1]
```diff
@@ -33,6 +33,36 @@ import (
 	"kubevirt.io/client-go/log"
 )
 
+// TODO this should be part of structs, instead of a global
+var DefaultOwnershipManager OwnershipManagerInterface = &OwnershipManager{uid: 107, gid: 107}
```
Comment (stu-gott) [inline pkg/ephemeral-disk-utils/utils.go]: hrm... well on systems I'm familiar with I recall UID 107 is qemu, but I don't think we can guarantee this is universal. recommend using `user.LookupId` if possible.

Comment (rmohr) [reply pkg/ephemeral-disk-utils/utils.go]: addressed

**Comment by rmohr** [comment]
Hm, the slirp issues are real. Could be because of av libvirt limitations.

**Comment by rmohr** [comment]
/hold

need to sort out the slirp issue.

**Comment by qinqon** [comment]
@rmohr looking...

**Comment by qinqon** [comment]
The qemu user networking params feel ok 
```
 -netdev user,id=default,net=10.0.2.0/24,dnssearch=kubevirt-test-default.svc.cluster.local,dnssearch=svc.cluster.local,dnssearch=cluster.local,hostfwd=tcp::80-:80
```

**Comment by qinqon** [comment]
Also at the virt-launcher 
```
The network interface type of default was changed to e1000 due to unsupported interface type by qemu slirp network
```

**Comment by qinqon** [comment]
Ok it work with wget

There is something fishy with curl version there:
```
[root@testvmih7s622crccmzw924gtxdcwqrdcdmn72xdkf7hh48tvwhvp2q /]# curl 127.0.0.1:80
curl: (1) Received HTTP/0.9 when not allowed
```
```
[root@testvmih7s622crccmzw924gtxdcwqrdcdmn72xdkf7hh48tvwhvp2q /]# curl --version
curl 7.66.0 (x86_64-redhat-linux-gnu) libcurl/7.66.0 OpenSSL/1.1.1d-fips zlib/1.2.11 brotli/1.0.7 libidn2/2.2.0 libpsl/0.21.0 (+libidn2/2.2.0) libssh/0.9.0/openssl/zlib nghttp2/1.39.2
Release-Date: 2019-09-11
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS brotli GSS-API HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz Metalink NTLM NTLM_WB PSL SPNEGO SSL TLS-SRP UnixSockets
```

**Comment by qinqon** [comment]
This works with curl

```
[root@testvmih7s622crccmzw924gtxdcwqrdcdmn72xdkf7hh48tvwhvp2q /]# curl --http0.9 127.0.0.1:80
Hello World!

```

**Comment by rmohr** [comment]
/close

in favour of #2909 and #2913

**Pull request closed without merging**